### PR TITLE
initial project proposal for emerald

### PR DESCRIPTION
**Pull request by Bartman250**
first draft of the proposal - needs review !

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (tkuhrt) [inline .gitignore]: Is this file required?

File: labs/emerald-performance-testharness.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Lab Name
```
Comment (tkuhrt) [inline labs/emerald-performance-testharness.md]: Should the file be named `labs/emerald-performance-test-harness.md`?  In other words, do you want a dash between `test` and `harness`. Remember, the name of the markdown file will be the name of the lab.

File: .gitignore (hunk 1 above)
Comment (Bartman250) [reply .gitignore]: Hello @tkuhrt  - apologies, I put in a .gitignore to ignore these files - it clearly did not work :(
I'll fix it and re-submit
